### DISCUSSION TITLE
2 minor issues (maybe for next update)

### DISCUSSION THREAD

**Discussion by kylemonti** [jomjol/AI-on-the-edge-device#3451] [Ideas] [answered]
First of all, big fan, been using 2 ESP32 with this software on and off for around a year. My wife sometimes likes to unplug my ESP32, and when they boot up, the first date they set is Jan 1970, which screws up the graphs. Could a future update be made to ignore the first couple of readings until the date & time are established? 

Edit: Removed 2nd question as it was obviously answered.

**Comment by SybexX** [comment, marked as answer]
![Unbenannt](https://github.com/user-attachments/assets/55f7c20e-193e-44e5-ae14-b219e8104aa6)

**Reply by kylemonti** [reply]
Thank you!

**Comment by caco3** [comment]
> Could a future update be made to ignore the first couple of readings until the date & time are established?

IMO it is more important to have the data with wrong date than no data at all.
Eg. if your NTP is setup wrong or does not work anymore, you would never get data which would be a pitty,
